CMakeLists.txt

```diff
@@ -1112,6 +1112,9 @@ if(SWIFT_INCLUDE_TOOLS)
   # https://bugs.swift.org/browse/SR-5975
   add_subdirectory(tools)
 
+  # Diagnostics information is only useful for the frontend compiler.
+  # It should only be included if the compiler is being built, which
+  # at the moment seems like if SWIFT_INCLUDE_TOOLS is defined.
```

Comment:
Isn't there also a dependency on swift-frontend that was mentioned?  I.e. we would fail to configure?

Reply:
Yup, crap, I forgot to add the one thing you requested originally :o

Reply:
No problem! Whats important is we get the right information here! = ).